code/datums/martial.dm

```diff
@@ -7,6 +7,7 @@
 	var/datum/martial_art/base = null // The permanent style
 	var/deflection_chance = 0 //Chance to deflect projectiles
 	var/help_verb = null
+	var/counter_prob = 0//probability to counter direct melee attacks
```

Comment:
I'm thinking I'm gonna just tear out all the countering shit, since it doesn't work, goof has given up on working on it, and it's probably unbalanced anyways. As far as I can tell, I can remove all the changes to this file.